homeassistant/components/netatmo.py

```diff
@@ -50,7 +64,10 @@ def setup(hass, config):
             "Please check your settings for NatAtmo API.")
         return False
 
-    for component in 'camera', 'sensor':
+    components = [component for component in ('camera', 'sensor')
```

Comment:
Ok, perfect.

I checked for cameras in the same way as the dependency just before it brokes using the raw data from the API.